c10/core/ScalarType.h

```diff
@@ -43,7 +44,8 @@ namespace c10 {
   _(c10::qint32, QInt32) /* 14 */                        \
   _(at::BFloat16, BFloat16) /* 15 */                     \
   _(c10::quint4x2, QUInt4x2) /* 16 */                    \
-  _(c10::quint2x4, QUInt2x4) /* 17 */
+  _(c10::quint2x4, QUInt2x4) /* 17 */                    \
+  _(c10::bits16, Bits16) /* 18 */
```

Comment:
So, even if you don't add support for the other types, I suggest adding them to the enum here, so that the numbering stays stable (and makes sense).

Reply:
I just updated the PR to contain all the other types :P